scripts/certificates/generate_device_certificate.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-if [[ $# -ne 2 ]]; then
-    echo "usage: $0 <project_id> <certificate_id>" >&2
+if [[ $# -lt 2 ]]; then
+    echo "usage: $0 <project_id> <certificate_id> [pem]" >&2
```

Comment:
I'm unfamiliar with the square-brackets syntax here (compared to the angle braces). Is this an argument in itself or is the user meant to replace that?

Reply:
Square brackets mean it's optional. The user should input "pem" exactly. If it's meant to be replaced and optional, then you would use both: `[<format>]`